docs/components/combobox-field/index.md

```diff
@@ -0,0 +1,311 @@
+# Combobox Field
+
+Provides a Toucan-styled combobox with filtering that builds on top of the Field component.
```

Comment:
Suggestions: Link to the field component, and maybe mention the benefits (error, labels, hint)

Reply:
Mind if I do this in a follow up PR? This is copy/pasta'd from the other docs, so I'd like to do them all in one go so it's consistent if possible!